crypto/bech32/bech32_test.go

```diff
@@ -0,0 +1,37 @@
+package bech32
+
+import (
+	"bytes"
+	"encoding/hex"
+	"testing"
+)
+
+func TestBench32EncodeDecode(t *testing.T) {
+	// bech32  -e -h tiov 746573742d7061796c6f6164
+	const enc = `tiov1w3jhxapdwpshjmr0v9jqymqq4y`
+
+	want, err := hex.DecodeString("746573742d7061796c6f6164")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	hrp, payload, err := Decode(enc)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !bytes.Equal(want, payload) {
```

Comment:
Please also test hrp (which should be tiov here, I believe)

Reply:
Done in https://github.com/iov-one/weave/pull/415/commits/b4fd343702ee9ed2b4918374384ed0c14c32b6e6